pkg/controller/jaeger/jaeger_controller.go

```diff
@@ -222,3 +231,48 @@ func (r *ReconcileJaeger) apply(jaeger v1.Jaeger, str strategy.S) error {
 
 	return nil
 }
+
+func containsFinalizer(jaeger *v1.Jaeger, finalizerID string) bool {
+	finalizers := jaeger.GetFinalizers()
+	for _, finalizer := range finalizers {
+		if finalizer == finalizerID {
+			return true
+		}
+	}
+	return false
+}
+
+func (r *ReconcileJaeger) handleFinalizer(deleted bool, jaeger *v1.Jaeger) error {
+	// Is marked for deletion?
+	var err error
+	if deleted {
+		jaeger.Logger().Debugf("%s is marked as deleted, executing finalizer", jaeger.Name)
```

Comment:
Use `jaeger.Logger().WithField("instance", jaeger.Name).Debug("instance marked as deleted, executing finalizer")`

Reply:
s the WithField("instance", jaeger.Name) necesary? I thought jaeger.Logger() returns an instance of the logger with those fields. At least is something saw on [1]

1) https://github.com/jaegertracing/jaeger-operator/blob/01bb012c81244a273cc7b391084aac4be0e4ac6a/pkg/apis/jaegertracing/v1/logger.go#L6